### ISSUE TITLE
Build error: cargo-install-update.rs does not exist

### ISSUE THREAD

**Issue by Seeker14491** [nabijaczleweli/cargo-update#26]
`cargo-update` fails to install.

```
[bash] ~: rustc --version
rustc 1.18.0-nightly (28a742997 2017-04-13)
[bash] ~: cargo install cargo-update
    Updating registry `https://github.com/rust-lang/crates.io-index`
  Installing cargo-update v0.8.0
   Compiling strsim v0.6.0
   Compiling void v1.0.2
   Compiling lazy_static v0.2.8
   Compiling ansi_term v0.9.0
   Compiling json v0.11.6
   Compiling vec_map v0.7.0
   Compiling unicode-segmentation v1.1.0
   Compiling unicode-width v0.1.4
   Compiling unreachable v0.1.1
   Compiling array_tool v0.3.4
   Compiling tabwriter v1.0.3
   Compiling libc v0.2.21
   Compiling regex-syntax v0.4.0
   Compiling serde v0.9.13
   Compiling bitflags v0.8.2
   Compiling quote v0.3.15
   Compiling embed-resource v1.1.1
   Compiling unicode-xid v0.0.4
   Compiling cargo-update v0.8.0
   Compiling semver-parser v0.7.0
   Compiling synom v0.11.3
   Compiling memchr v1.0.1
   Compiling term_size v0.3.0
   Compiling rand v0.3.15
   Compiling syn v0.11.10
   Compiling thread-id v3.0.0
   Compiling utf8-ranges v1.0.0
   Compiling thread_local v0.3.3
   Compiling aho-corasick v0.6.3
   Compiling semver v0.6.0
   Compiling atty v0.2.2
   Compiling clap v2.23.2
   Compiling lazysort v0.1.1
   Compiling toml v0.3.2
   Compiling regex v0.2.1
   Compiling serde_codegen_internals v0.14.2
   Compiling serde_derive v0.9.13
error: couldn't read ".cargo/registry/src/github.com-1ecc6299db9ec823/cargo-update-0.8.0/src/bin/cargo-install-update.rs": No such file or directory (os error 2)

error: Could not compile `cargo-update`.
Build failed, waiting for other jobs to finish...
error: failed to compile `cargo-update v0.8.0`, intermediate artifacts can be found at `/tmp/cargo-install.B0YOp9aGjRxe`

Caused by:
  build failed
```

**Comment by nabijaczleweli** [comment]
Seems to me like nightly now assumes binary name as source name?
Try installing newest `master` for fix.

**Comment by Drakulix** [comment]
master fixed it for me. Could you do a crates.io release?

**Comment by nabijaczleweli** [comment]
Released in [`v0.8.1`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v0.8.1).
